site/gatsby-site/src/templates/post.js

```diff
@@ -13,11 +13,13 @@ import { LocalizedLink } from 'plugins/gatsby-theme-i18n';
 import { useLayoutContext } from 'contexts/LayoutContext';
 
 export default function Post(props) {
-  let {
-    data: { mdx, enMdx },
+  const {
+    data: { mdx: translatedMdx, enMdx },
```

Comment:
I like the `const` change, but the other stuff seems unnecessary 🤔

Reply:
The reason for this change is to improve the variable's name clarity using the word `translated`. I will change the code to use a comment instead.